### DISCUSSION TITLE
Is there any way to search/find a specific string, and get its cell/row/column, from a worksheet?

### DISCUSSION THREAD

**Discussion by dongyi-cai-windsab** [ClosedXML/ClosedXML#1994] [Q&A] [answered]
As title suggested. I need a "Find" or "Search" feature.

**Comment by igitur** [comment, marked as answer]
https://github.com/ClosedXML/ClosedXML/blob/78391cdd980010e0639522345b5b0f663cb8b9ed/ClosedXML/Excel/Ranges/IXLRangeBase.cs#L85

**Reply by dongyi-cai-windsab** [reply]
Thanks
